customer-node/service/ticketsService.js

```diff
@@ -0,0 +1,19 @@
+const {
+  findTicketsByCustomer, createTicket, findTicketById, deleteTicket,
+} = require('../data_access/ticketda');
+
+const getTicketsByCustomerId = (customerId) => findTicketsByCustomer(customerId);
```

Comment:
Check if a customer with such an Id exists before querying for tickets

Reply:
done